deploy/template.yaml

```diff
@@ -45,6 +40,13 @@ Parameters:
       The number of core front ecs tasks to run.
     Type: Number
 
+Conditions:
+  IsNotDevelopment: !Or
+    - !Equals [ !Ref Environment, build ]
+    - !Equals [ !Ref Environment, staging ]
+    - !Equals [ !Ref Environment, integration ]
+    - !Equals [ !Ref Environment, production ]
+
```

Comment:
could not get a wildcard to work for `dev*`, so had to use this inverse approach.